packages/apps/shopify-api/lib/clients/admin/__tests__/rest_client.test.ts

```diff
@@ -594,7 +594,7 @@ describe('REST client', () => {
     await client.get({path: '/url/path'});
 
     // first call to .log is .debug with package and runtime info during initialization
-    expect(shopify.config.logger.log).toHaveBeenCalledTimes(2);
+    expect(shopify.config.logger.log).toHaveBeenCalledTimes(3);
```

Comment:
All these tests had to be updated because we added another log. We have the options of either updating these log counts or we could turn off the `v10_lineItemBilling` for these tests. (These tests don't test billing)

Reply:
It'd be great if we didn't depend on the count here at all, but turning the flag off sounds reasonable too, because then we won't have to change them back later.